### PR TITLE
Joins update

### PR DESCRIPTION
**Pull request by ethanwhite**

### CODE REVIEW

File: materials/sql_joins.md [hunk 1]
```diff
@@ -9,6 +9,24 @@ language: SQL
...
+* It is often not efficient to include all information of interest in a single
+table.
+* Redundant information makes it more difficult to update or revise data.
+    * If something changes we want to be able to change it in one place, not
+    hundreds of places.
+* Use multiple tables
+* Each table contains a single kind of information
+    * `surveys`: information about individuals
+    * `species`: information about species
+	* `plots`: information about plots
```
Comment (brymz) [inline materials/sql_joins.md]: indent issue.

**Comment by brymz** [comment]
This will cause another merge conflict with #477.

**Comment by ethanwhite** [comment]
Rebased against `gh-pages`. Fixed indent.

**Comment by ethanwhite** [comment]
@brymz - A note about rebase, which I just used to fix the merge issue, as part of your ongoing git education. Sometimes, rebasing can make fixing these sorts of conflicts automatically. In this case I did the following from the `joins-update` branch:

```
git fetch upstream
git rebase upstream/gh-pages 
```

And I was already to go with no merge conflicts to resolve. Rebasing works by replaying commits one at a time. Of course it can still result in conflicts, which you solve in the same way as with merging.

**Comment by ethanwhite** [comment]
Also, just updated this PR to make the first joins example differ from the Joins 0 exercise (great minds think alike, at least if they are my mind working on the exercise and lecture on two different days... oops).

**Comment by brymz** [comment]
> About `rebase`

How is this different from `git pull upstream gh-pages` from the local branch? `pull` sets local first with merge to `gh-pages`, while `rebase` sets `gh-pages` first and restructures as needed?

File: materials/sql-joins.md [hunk 2]
```diff
@@ -17,33 +35,39 @@ language: SQL
...
 FROM surveys
-JOIN species ON surveys.species_id = species.species_id;
+JOIN species ON surveys.plot_id = plots.plot_id
 ```
+
 * This query selects `year`, `month`, and `day` from `surveys` and 
-`genus` and `species` from the `species` table.
-    * The query links the `species_id` from `surveys` with `species_id` from `species`.
+`plot_type` from the `plots` table.
+    * The query links the `species_id` from `surveys` with `plot_id` from `plots`.
```
Comment (brymz) [inline materials/sql-joins.md]: '... links the `plot_id` from `surveys` ...'

**Comment by ethanwhite** [comment]
Fixed.

> How is this different from git pull upstream gh-pages from the local branch?

Instead of trying to combine the end points in their current state, it replay's the changes sequentially.

**Pull request merged**